src/main/host/syscall_handler.c

```diff
@@ -506,7 +506,7 @@ SysCallReturn syscallhandler_make_syscall(SysCallHandler* sys,
     } else if (_syscallhandler_wasBlocked(sys)) {
         /* We were but are no longer blocked on a syscall. Make
          * sure any previously used listener timeouts are ignored.*/
-        _syscallhandler_setListenTimeout(sys, NULL);
+        _syscallhandler_setListenTimeout(sys, NULL, TIMEOUT_ABSOLUTE);
```

Comment:
Isn't `TIMEOUT_RELATIVE` consistent with the code before, rather than `TIMEOUT_ABSOLUTE`?

---

Using `TIMEOUT_ABSOLUTE ` will cause us to pass `flags=TFD_TIMER_ABSTIME` to `timer_setTime`, whereas before we were passing `flags=0` to `timer_setTime`.

This will cause us to take the `if` statement in `_timer_setCurrentTime`, whereas before we took the `else` statement.

I *think* the end result is the same, but we should probably be consistent and use `TIMEOUT_RELATIVE` instead.

Reply:
Done